### DISCUSSION TITLE
Calendar

### DISCUSSION THREAD

**Discussion by ali-benny** [csunibo/csunibo.github.io#32] [Domande]
Ciao a tutti! Dall' csunibo/dynamik#57 si è reso necessario creare una nostra versione custom made del visualizzatore online di unibo-calendar. 

Vi chiedo: dove lo mettiamo? In quale repo? sarebbe carino avere lo stesso stile grafico di dynamik secondo me.
Attualmente ho una versione fatta lì che si visualizza come:
<img width="826" alt="image" src="https://github.com/csunibo/csunibo.github.io/assets/74602443/118f592a-5e87-43e3-b265-69c20c9188cd">

**Comment by foxyseta** [comment]
Io direi una repo nuova, distinta da https://github.com/csunibo/unibo-calendar. La domanda a sto punto però è: non stiamo creando un clone del calendario sul sito unibo? Non possiamo semplicemente offrici di fornire le integrazioni per le applicazioni calendario?

**Reply by ali-benny** [reply]
Nella nostra versione di https://github.com/csunibo/unibo-calendar c'era la possibilità di visualizzarlo online e @csunibo/informatica-1 lo trovavano comodo da quanto ho capito

**Reply by foxyseta** [reply]
Cioè che ha funzionalità in più rispetto a quello su unibo.it?

**Reply by samuelemusiani** [reply]
In realtà non usiamo più quel calendario, ma utilizziamo direttamente quello di Eyad: https://github.com/VaiTon/UniboCalendar

**Reply by ali-benny** [reply]
> Cioè che ha funzionalità in più rispetto a quello su unibo.it? @foxyseta 

nessuna, ma era già presente come opzione nella repo di @VaiTon e per sistemare l'orario si doveva fare custom.
A questo punto però se vogliamo aggiungere delle funzionalità si può, come ad esempio mandare notifiche(?) la trovo un po' inutile ma è per dirne una

**Reply by foxyseta** [reply]
Si può considerare di mantenere la GUI del calendario solo se ha effettivamente funzionalità utii in più rispetto a unibo.it. Sennò potrebbe essere un inutile clone. Lo scopo originale dell'applicativo per me è fornire ICS/ICAL e simili. Ovviamente questa è una mia opinione quindi si può comunque procedere senza problemi.

**Reply by VaiTon** [reply]
E' utile perché è integrata, ma non saprei dire quanto il gioco valga la candela. Io la accorperei in dynamik TBH

**Reply by foxyseta** [reply]
integrata cioè non devi lasciare il dominio per visitarla? Perché a me va bene lasciarla, però secondo la stessa politica dovremmo clonare ogni servizio unibo.it dentro il nostro dominio senza aggiungere funzionalità solo perché "è integrato quindi utile".

**Reply by VaiTon** [reply]
Su questo hai ragione. Sinceramente a me il calendario di unibo non mi fa impazzire per nulla, quindi potremmo farlo meglio secondo me. 

Detto ciò, io non avrei il tempo di seguire questo mini-progetto, quindi se qualcuno vuole e riesce a farlo perché no dopotutto! Altrimenti si crea un link da calendar al calendario di unibo.it e bon :)

**Reply by foxyseta** [reply]
Concordo

**Comment by foxyseta** [comment]
Al momento stiamo usando https://github.com/simonrob/online-ics-feed-viewer, che immagino risolva il problema originale con UTC. Questa discussione quindi si può segnare come risolta, dal momento che non abbiamo un motivo vero e proprio per reinventare la ruota al momento?

**Reply by VaiTon** [reply]
Direi di si :)

**Reply by VaiTon** [reply]
cc @ali-benny @samuelemusiani

**Reply by samuelemusiani** [reply]
Per me è risolta

**Reply by ali-benny** [reply]
va bene
